### PR TITLE
Constant SAMPLE_TIME replaced with auto-generated

### PR DESCRIPTION
**Pull request by alexey-kravtsov**

### CODE REVIEW

File: src/test/java/com/axibase/tsd/api/method/series/SeriesInsertTest.java [hunk 1]
```diff
@@ -513,7 +513,7 @@ public void testMillisecondsUnsupported() throws Exception {
         String metricName = "m-iso-13";
 
         Series series = new Series(entityName, metricName);
-        series.addSamples(Sample.ofRawDateInteger(Mocks.MILLS_TIME.toString(), 0));
+        series.addSamples(Sample.ofRawDateInteger("1470009600000", 0));
```
Comment (vtols) [inline src/test/java/com/axibase/tsd/api/method/series/SeriesInsertTest.java]: It is a bit strange that we use timestamp here. How does it work?
https://github.com/axibase/atsd/blob/master/api/data/series/insert.md#value-object

Comment (alexey-kravtsov) [reply src/test/java/com/axibase/tsd/api/method/series/SeriesInsertTest.java]: This test checks for incorrect date string. Timestamp is passed as a date and test waits for error message

Comment (vtols) [reply src/test/java/com/axibase/tsd/api/method/series/SeriesInsertTest.java]: OK

File: src/test/java/com/axibase/tsd/api/util/Mocks.java [hunk 2]
```diff
@@ -4,14 +4,18 @@
...
 import java.util.Date;
 import java.util.Map;
 
 public class Mocks {
     private static final TestNameGenerator NAME_GENERATOR = new TestNameGenerator();
 
-    public static final String ISO_TIME = "2017-08-01T00:00:00.000Z";
+    // generating date with current year because ATSD does not store messages older than year
+    public static final String ISO_TIME =
+            String.format("%s-08-01T00:00:00.000Z",
```
Comment (vtols) [inline src/test/java/com/axibase/tsd/api/util/Mocks.java]: Maybe it is better to substitute the date with current year right in place where it is necessary?

**Comment by vtols** [comment]
You can leave ISO_TIME unchanged

**Pull request merged**